File: src/main/scala-graphql/io/github/graphql/model/DeploymentStateTO.scala
package io.github.graphql.model

import com.fasterxml.jackson.core.`type`.TypeReference

@javax.annotation.Generated(
    value = Array("com.kobylynskyi.graphql.codegen.GraphQLCodegen"),
    date = "2021-03-06T19:48:00+0800"
)
object DeploymentStateTO extends Enumeration {

    type DeploymentStateTO = Value

    val ABANDONED: Value = Value("ABANDONED")
    val ACTIVE: Value = Value("ACTIVE")
    val DESTROYED: Value = Value("DESTROYED")
    val ERROR: Value = Value("ERROR")
    val FAILURE: Value = Value("FAILURE")
    val INACTIVE: Value = Value("INACTIVE")
    val IN_PROGRESS: Value = Value("IN_PROGRESS")
    val PENDING: Value = Value("PENDING")
    val QUEUED: Value = Value("QUEUED")

}

class DeploymentStateTOTypeRefer extends TypeReference[DeploymentStateTO.type]

File: src/main/java-graphql/io/github/graphql/model/DeploymentStateTO.java
package io.github.graphql.model;

@javax.annotation.Generated(
    value = "com.kobylynskyi.graphql.codegen.GraphQLCodegen",
    date = "2021-03-06T19:47:47+0800"
)
public enum DeploymentStateTO {

    ABANDONED("ABANDONED"),
    ACTIVE("ACTIVE"),
    DESTROYED("DESTROYED"),
    ERROR("ERROR"),
    FAILURE("FAILURE"),
    INACTIVE("INACTIVE"),
    IN_PROGRESS("IN_PROGRESS"),
    PENDING("PENDING"),
    QUEUED("QUEUED");

    private final String graphqlName;

    private DeploymentStateTO(String graphqlName) {
        this.graphqlName = graphqlName;
    }

    @Override
    public String toString() {
        return this.graphqlName;
    }

}
